src/utils.js

```diff
@@ -17,3 +17,17 @@ export function buildRegExpFromDelimiters(delimiters) {
   const escapedDelimiterChars = escapeRegExp(delimiterChars);
   return new RegExp(`[${escapedDelimiterChars}]+`);
 }
+
+/**
+ * Escape text to be used in HTML.
+ * @param {string} text to be escaped
+ * @returns {string} HTML
+ */
+export function escapeHTML(text) {
```

Comment:
`lodash` is already added in this library so can we use `escape` in lodash so that the custom escape can be omitted ? in that case it can be directly used `Suggestion.js`